### PR TITLE
Adding MIT open-source license to the project

### PR DESCRIPTION
**Pull request by notkaramel**
As the title suggested, I've added an MIT license to the project as it is now open-source.

I want to make sure the details I put on the license looks good for the project, specifically on line 3 of the license:
- Year: **2024**
- Copyright holder: **The ECHOES Research Project**

Please review, and feel free to merge the PR if the license looks good!

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2024 The ECHOES Research Project
```
Comment (martha-thomae) [inline LICENSE:3]: Echoes from the Past: Unveiling a Lost Soundscape with Digital Analysis.

Year is fine.

**Review by martha-thomae — COMMENTED** [review]
I just left you the comment from Elsa's email. Thanks, Antoine. Please make sure that this project is open-access and with no restrictions.

File: LICENSE (hunk 1 above)
Comment (notkaramel) [reply LICENSE]: I just updated the project name!

Comment (notkaramel) [reply LICENSE]: Just want to make sure, "open-access and **with no restrictions**" means that it's viewable to the public and everyone can clone/fork to contribute, right? I wouldn't let non-maintainer/collaborator have direct push privilege :/

**Comment by elsinhadl** [comment]
Hi there,
My idea of open access for this kind of ECHOES output is that people can
freely reuse what we do (copy & edit at their will) but, at the same time,
only you two can alter/edit what we create and, as Antoine says, have
direct push privilege.
Thank you!
Elsa

*________________________________________________________*

PI of *Echoes from the Past: Unveiling a Lost Soundscape with Digital
Analysis
<https://sites.google.com/fcsh.unl.pt/elsadeluca/echoes-research-project> *
(2022.01957.PTDC)

Recent publication: *The Materiality of Sound in ChantManuscripts in the
West
<https://www.brepols.net/products/IS-9782503606149-1?fbclid=IwAR2gxS-ucQAfNZEgO9rjGSbBUcpXbSKBkictNRCHw5uqWjfIDCw0oyhYpso>*,
Brepols 2023


CESEM - Centre for the Study of the Sociology and Aesthetics of Music
IN2PAST – Associate Laboratory for Research and Innovation in Heritage,
Arts, Sustainability and Territory
School of Social Sciences and Humanities
NOVA University of Lisbon

https://sites.google.com/fcsh.unl.pt/elsadeluca/ |
*https://cesem.fcsh.unl.pt/en/pessoa/elsa-de-luca-2/
<https://cesem.fcsh.unl.pt/en/pessoa/elsa-de-luca-2/> *



Antoine Phan ***@***.***> escreveu (quinta, 25/01/2024 à(s)
03:06):

File: LICENSE (hunk 1 above)
Comment (notkaramel) [reply LICENSE]: Got it, thanks!

**Comment by notkaramel** [comment]
Thank you for the clarification, I will merge this PR now!

**Pull request merged**